jdaviz/configs/imviz/plugins/catalogs/catalogs.py

```diff
@@ -303,16 +292,19 @@ def _on_is_active_changed(self, *args):
 
     def plot_selected_points(self):
         selected_rows = self.table.selected_rows
+        if len(selected_rows) == 0 or 'x_coord' not in selected_rows[0]:
```

Comment:
when would x_coord not be populated?  Could it be in `selected_rows[0]` but not in another row?

Reply:
That was part of the bug I was seeing earlier. Rows were being added in the first `if catalog == 'SDSS'` without `x_coord` and then again later in the file with `x_coord`. These lines can safely be removed.